CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ All notable changes to this project will be documented in this file.
 
 ### Changed
 - Testing framework changed from `nose` to `pytest`.
+- Updated `hera_qm` test call on circleci
```

Comment:
I don't think we need this in the changelog, it doesn't have anything to do with pyuvdata code or functionality.

Reply:
where's the delete forever button?